nav-bar.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Nav Bar</title>
+    <link rel="stylesheet" href="nav-bar.css">
+</head>
+<body>
+    <div class="nav-bar">
+        <div class="menu-item">
+            <img src="colors.jpg" width="50px" height="50px">
+        </div>
+        <div class="menu-item">
```

Comment:
Toate link-urile de navigare le puteai pune intr-un parinte < nav > si in interior un < ul > si toate div-uriele sa le pui intr-un parinte < li >
Ca HTML-ul sa fie construit cat mai semantic posibil folosing de toate elementele specifice pentru o bara de navigare cat si pentru o lista de link-uri de navigare